### DISCUSSION TITLE
[Projects Beta] Standard Workflow should move issues when linked PRs change

### DISCUSSION THREAD

**Discussion by arnodirlam** [community/community#10402] [Projects and Issues]
Hi there,

we prefer adding issues to the project board, as there's already a column "linked PRs" available 👍 

However, when workflows such as "Code changes requested" or "Code review approved" occur for a PR, I'd expect the linked issue in the Project to have the configured Status set. This is currently not the case; only for PRs themselves, not issues with linked PRs. I'd appreciate implementing that!🙏

Cheers, Arno

**Comment by edwardoranj** [comment]
This is a really useful feature to make the issue focused boards automated!
+1 on this
